### PR TITLE
Update Ruby to 2.7.5

### PR DESCRIPTION
**Pull request by jkempster34**
Update Ruby to 2.7.5

### CODE REVIEW

**Review by leoapost — CHANGES_REQUESTED** [review]
Can we also update the README?

**Comment by jkempster34** [comment]
> 

Done 👍

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ We also have markdown pages within the `doc` folder of this git repo
 
 ## Prerequisites
 
-- Ruby 2.7.4 - easiest with rbenv and ruby-build
+- Ruby 2.7.5 - easiest with rbenv and ruby-build
   - `brew install rbenv`
   - `brew install ruby-build`
   - `rbenv install 2.7.4`
```
Comment (leoapost) [inline README.md:25]: One last mention I believe in this line

**Review by leoapost — APPROVED** [review]
:100:

**Pull request merged**